### ISSUE TITLE
swagger validation for new contact is not up to date

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#528]
I got the following error when posting new contact:

````
POST http://localhost:4000/api/v1/contacts
{"name_prefix":"","given_name":"azd","family_name":"","name_suffix":"","organizations":[],"identities":[],"emails":[]}
```

```
{
	"errors": [{
		"code": 400,
		"description": "The server could not comply with the request since it is either malformed or otherwise incorrect.",
		"component": "Unknown",
		"values": 400,
		"message": "Additional properties are not allowed (u'identities' was unexpected)\n\nFailed validating u'additionalProperties' in schema:\n    {u'additionalProperties': False,\n     u'properties': {u'additional_name': {u'type': u'string'},\n                     u'emails': {u'items': {u'additionalProperties': False,\n                                            u'properties': {u'address': {u'type': u'string'},\n                                                            u'is_primary': {u'type': u'boolean'},\n                                                            u'label': {u'type': u'string'},\n                                                            u'type': {u'type': u'string'}},\n                                            u'required': [u'address'],\n                                            u'type': u'object'},\n                                 u'type': u'array'},\n                     u'family_name': {u'type': u'string'},\n                     u'given_name': {u'type': u'string'},\n                     u'groups': {u'items': {u'type': u'string'},\n                                 u'type': u'array'},\n                     u'ims': {u'items': {u'type': u'object'},\n                              u'type': u'array'},\n                     u'infos': {u'type': u'object'},\n                     u'name_prefix': {u'type': u'string'},\n                     u'name_suffix': {u'type': u'string'},\n                     u'organizations': {u'items': {u'type': u'object'},\n                                        u'type': u'array'},\n                     u'tags': {u'items': {u'type': u'object'},\n                               u'type': u'array'}},\n     u'type': u'object'}\n\nOn instance:\n    {u'emails': [],\n     u'family_name': u'',\n     u'given_name': u'azd',\n     u'identities': [],\n     u'name_prefix': u'',\n     u'name_suffix': u'',\n     u'organizations': []}",
		"property": "RequestValidationError",
		"type": "Bad Request"
	}]
}
```

**Comment by iamdey** [comment]
blocking bug

**Comment by sapiens-sapide** [comment]
Fix by commit #651fad9
